plugins/rule_engines/irods_rule_language/include/irods/private/re/restructs.hpp

```diff
@@ -333,7 +333,7 @@ ExprType *newTupleTypeVarArg( int arity, int vararg, ExprType **paramTypes, Regi
 ExprType *newSimpType( NodeType t, Region *r );
 ExprType *newErrorType( int errcode, Region *r );
 ExprType *newIRODSType( const char *name, Region *r );
-ExprType *newFlexKind( int arity, ExprType **typeArgs, Region *r );
+/*ExprType *newFlexKind( int arity, ExprType **typeArgs, Region *r );*/
```

Comment:
same?

Reply:
From the PR OP:
> In some cases I have commented out function declarations instead of removing them, as the function definition *is* still present in the code, but commented out. I'm not going to remove these declarations or commented code without getting more familiar with the rule engine plugin or knowing why the code was commented out in the first place rather than removed. But that's a problem for future Markus, or whoever ends up picking up this refactor project in the future. I've got other priorities right now.